packages/react/types/tests-css.tsx

```diff
@@ -32,8 +32,8 @@ css`
 // $ExpectError
 css(() => 'height: 300px;')
 
-// $ExpectError
 css`
   position: relative;
-  flexgrow: ${() => 20};
+  flexgrow: ${// $ExpectError
```

Comment:
I think this might have broken `yarn test:typescript` task.

Reply:
ah, the comment was supposed to go above the arrow function, but prettier moved it during the pre-commit hook. I've added a comment to tell prettier to ignore this line.